src/components/generic/course-card.tsx

```diff
@@ -85,12 +144,38 @@ const CourseCard: React.FC<Props> = ({
    */
   return (
     <>
-      <PaperCard elevation={6} sx={{ width: 700, cursor: "pointer" }} onClick={() => setDialogOpen(true)}>
+      <PaperCard elevation={6} sx={{ width: 700, cursor: "pointer" }} onClick={() => !(hoveredUp || hoveredDown) && setDialogOpen(true)}>
         <Stack spacing={2}>
-          <Stack>
+          <Stack direction="row" alignItems="center" justifyContent="space-between">
             <Typography variant="h3">{course.name}</Typography>
-            <Typography variant="body2">{course.code}</Typography>
+            <Stack direction="row" alignItems="center" spacing={1}>
+              {!dislikePressed && (
+                <IconButton disabled={likePressed} onClick={() => course.desc && handleLike(course.desc)}>
+                  <ThumbUpRounded
+                    sx={{
+                      cursor: "pointer",
+                      color: hoveredUp ? "#4caf50" : "inherit"
```

Comment:
Would it better to use '''theme.palette.success.light''' than a hardcoded color

Reply:
You are right! Much appreciated the suggestion! Thank you